hack/lib/util.sh

```diff
@@ -461,7 +455,12 @@ kube::util::ensure_godep_version() {
 kube::util::ensure_no_staging_repos_in_gopath() {
   kube::util::ensure_single_dir_gopath
   local error=0
-  for repo in $(ls ${KUBE_ROOT}/staging/src/k8s.io); do
+  for repo_file in ${KUBE_ROOT}/staging/src/k8s.io/*; do
```

Comment:
quote the `"${KUBE_ROOT}"` bit before the glob?

Reply:
Ok.